osu.Game/Overlays/Comments/ReportCommentPopover.cs

```diff
@@ -0,0 +1,111 @@
+            {
+                Direction = FillDirection.Vertical,
+                Width = 500,
+                AutoSizeAxes = Axes.Y,
+                Spacing = new Vector2(7),
+                Children = new Drawable[]
+                {
+                    new OsuSpriteText
+                    {
+                        Origin = Anchor.TopCentre,
+                        Anchor = Anchor.TopCentre,
+                        Text = UsersStrings.ReportTitle(comment.Comment.User?.Username ?? comment.Comment.LegacyName!),
+                        Font = OsuFont.Torus.With(size: 25),
+                        Margin = new MarginPadding { Bottom = 10 }
+                    },
+                    new OsuSpriteText
+                    {
+                        Origin = Anchor.TopCentre,
+                        Anchor = Anchor.TopCentre,
+                        Text = UsersStrings.ReportReason,
+                        Font = OsuFont.Torus.With(size: 20),
+                    },
+                    reason = new OsuEnumDropdown<CommentReportReason>
+                    {
+                        RelativeSizeAxes = Axes.X
+                    },
+                    new OsuSpriteText
+                    {
+                        Origin = Anchor.TopCentre,
+                        Anchor = Anchor.TopCentre,
+                        Text = UsersStrings.ReportComments,
+                        Font = OsuFont.Torus.With(size: 20),
+                    },
+                    info = new OsuTextBox
+                    {
+                        RelativeSizeAxes = Axes.X,
+                        PlaceholderText = UsersStrings.ReportPlaceholder
+                    },
+                    new Container
+                    {
+                        RelativeSizeAxes = Axes.X,
+                        AutoSizeAxes = Axes.Y,
+                        Child = shaker = new ShakeContainer
+                        {
+                            RelativeSizeAxes = Axes.X,
+                            AutoSizeAxes = Axes.Y,
+                            Child = new RoundedButton
+                            {
+                                Origin = Anchor.TopCentre,
+                                Anchor = Anchor.TopCentre,
+                                Width = 200,
+                                BackgroundColour = colours.Red3,
+                                Text = UsersStrings.ReportActionsSend,
+                                Action = send,
+                                Margin = new MarginPadding { Bottom = 5, Top = 10 },
+                            }
+                        }
+                    }
+                }
+            };
+        }
+
+        private void send()
+        {
+            string infoValue = info.Current.Value;
+            var reasonValue = reason.Current.Value;
+
+            if (string.IsNullOrWhiteSpace(infoValue))
+            {
+                shaker.Shake();
+                return;
+            }
```

Comment:
Not sure if it's only me, but I would prefer button displaying in disabled state rather than shaking.